### DISCUSSION TITLE
I want to insert multiple rows into an existing table.

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#1854] [Questions] [answered]
I want to insert multiple rows into an existing table.
Is there any support for this in the dashboard UI?

**Comment by silentworks** [comment, marked as answer]
@ru88s You can do this with the supabase-js library. Or if you are looking to populate the database with a CSV or an existing database dump, you can connect directly to the database using [TablePlus](https://tableplus.com/), [Beekeeper Studio](https://www.beekeeperstudio.io/) or whichever client you want to use.

You can find the database settings under **settings/database**.

In the case of using the supabase-js library, the code would look similar to below:

```js
await supabase.from('table_name').insert([
  { fieldName: value }, // individual row
  { fieldName: value },
])
```

**Reply by awalias** [reply]
additionally you can insert multiple rows using the SQL editor in the dashboard like:

```sql
INSERT INTO 
    links (url, name)
VALUES
    ('https://www.google.com','Google'),
    ('https://www.yahoo.com','Yahoo'),
    ('https://www.bing.com','Bing');
```

**Reply by ru88s** [reply]
@silentworks, @awalias, Thank you so much!
